### ISSUE TITLE
DATASET! What are the main types of things on your TOD dataset!

### ISSUE THREAD

**Issue by KeyPisces** [chrisdxie/uois#4]
Hello, first of all thanks for sharing your amazing job. I want to know What are the main types of things on your TOD dataset! and  if you can provide the dataset you used for training (TOD and set of OID data) and testing.
Thanks in advance ;)

**Comment by chrisdxie** [comment]
Hi @KeyPisces, you can see in [data_loader.py](https://github.com/chrisdxie/uois/blob/master/src/data_loader.py#L193) that the main items the dataloader returns are: RGB image, back-projected depth image (XYZ), foreground labels, and direction labels. The dimensions of each tensor are documented in the comments. Please take a look at the code for more details.

Unfortunately I cannot provide my training dataset (TOD) at this time. However, [OID](https://storage.googleapis.com/openimages/web/index.html) is a public dataset and you can download the segmentations. You will probably have to do a bit of processing to get them in the form that is useable from the dataloader, or you can just edit the dataloader.
